requirements/edx/testing.txt

```diff
@@ -68,7 +68,6 @@ attrs==22.1.0
     #   edx-ace
     #   jsonschema
     #   openedx-events
-    #   outcome
```

Comment:
Spurious removal but I guess it doesn't matter. Not sure why the upgrade job and our local envs differ on this one.